control/autopilot/include/autopilot/autopilot_inl.h

```diff
@@ -877,16 +899,27 @@ quadrotor_common::ControlCommand AutoPilot::start(
           + start_land_velocity_
               * (timeInCurrentState() - start_idle_duration_);
       reference_state_.velocity.z() = start_land_velocity_;
+      if(timeInCurrentState() < start_idle_duration_ +
```

Comment:
This is a hack since the velocity does not change with acceleration over time. But as long as it works for both controllers I can live with it ;) The nicest way would be if it would be a real trapezoidal velocity profile during the start maneuver.

Reply:
I'm gonna change that, test in simulation and request an approve again

Reply:
I opened an issue regarding this (#64). Feel free to adress it in a new pull request.

Reply:
already done here